app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -421,9 +425,9 @@ fun pictrsImageThumbnail(src: String, thumbnailSize: Int): String {
     }
 
     val host = split[0]
-    val path = split[1]
+    val path = split[1].replaceAfter('?', "")
```

Comment:
What's going on here?

Reply:
It was generating paths like this:

`https://server/path?thumbnail=x?thumbnail=y`

So I cut off the query params before the code later adds it back on. I could've done it more elegantly, but since I was just writing a bunch of tests, I wanted to keep the non-test code changes minimal. I intend to go back through later and clean up the code a bit (I'd like to separate the utils that don't touch UI stuff from the rest).

Reply:
Added a comment, this is resolved.